Commands/BowerInstallCommand.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Pingpong\Modules\Commands;
+
+use Illuminate\Console\Command;
+use Pingpong\Modules\Module;
+use Pingpong\Modules\Repository;
+
+class BowerInstallCommand extends Command
+{
+
+    /**
+     * @var string
+     */
+    protected $signature = 'module:bower-install {--production}';
+
+    /**
+     * @var string
+     */
+    protected $description = 'Runs `bower install` on all modules, where applicable';
+
+    /**
+     * @var Repository;
+     */
+    protected $modules;
+
+    /**
+     * BowerInstallCommand constructor.
+     * @param Repository $module
+     */
+    public function __construct(Repository $modules)
+    {
+        parent::__construct();
+
+        $this->modules = $modules;
+    }
+
+
+    public function fire()
+    {
+        // Check bower is installed
+        if (!$this->isBowerInstalled()) return;
+
+        foreach($this->modules->all() as $module) {
+            $module = $this->modules->findOrFail($module);
+
+            $this->handleModuleBowerInstall($module);
+        }
+    }
+
+
+    protected function handleModuleBowerInstall(Module $module)
+    {
+        // Check if bower.json exists
```

Comment:
Agreed, but not concerning the error message when a `bower.json` file wasn't found. Instead, let's do a simple `$this->info()` informing the user. Not all modules have bower-installed assets, and not using bower shouldn't result in an erroneous message.